scripts/update_schemas_version.py

```diff
@@ -0,0 +1,29 @@
+import sys
+from fileinput import FileInput
+
+import requests
+
+from requests.exceptions import RequestException
+
+
+try:
+    response = requests.get(
+        "https://api.github.com/repos/ONSdigital/eq-questionnaire-schemas/releases"
```

Comment:
You can visit https://api.github.com/repos/ONSdigital/eq-questionnaire-schemas/releases/latest to get the latest release rather than all.

Reply:
Thanks👍